pages/common/kdenlive.md

```diff
@@ -0,0 +1,28 @@
+# kdenlive
+
+> KDE's non-linear video editor.
+> More information: <https://manned.org/kdenlive>.
+
+- Start the video editor:
+
+`kdenlive`
+
+- Open a specific file:
+
+`kdenlive {{path/to/file}}`
```

Comment:
Does this open anything other than `.kdenlive` projects?

Reply:
I don't know actually because I don't use `kdenlive`. 🤔

Reply:
Yeah, I checked it. It opens only kdenlive project files.